Samples/MNIST/UWP/cs/MainPage.xaml

```diff
@@ -5,36 +5,42 @@
     xmlns:local="using:MNIST_Demo"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-    mc:Ignorable="d" Width="731.195" Height="496">
+    mc:Ignorable="d" Width="768.333" Height="519.111">
 
     <Grid
         x:Name="parentGrid"
-        Margin="40">
+        Margin="40,40,40,10" HorizontalAlignment="Stretch" VerticalAlignment="Stretch">
         <Grid.RowDefinitions>
             <RowDefinition Height="70" />
             <RowDefinition Height="336" />
+            <RowDefinition Height="60" />
         </Grid.RowDefinitions>
         <Grid.ColumnDefinitions>
-            <ColumnDefinition Width="336" />
+            <ColumnDefinition Width="25.778" />
+            <ColumnDefinition Width="37.222"/>
+            <ColumnDefinition Width="272.889"/>
             <ColumnDefinition Width="336" />
         </Grid.ColumnDefinitions>
 
         <TextBlock Text="Handwritten Digit:"
                FontSize="36"
                VerticalAlignment="Center"
-               Grid.Row="0"
-               Grid.Column="0" />
+               Grid.Column="0" Grid.ColumnSpan="3" />
         <TextBlock Text="Result:"
                FontSize="36"
                VerticalAlignment="Center"
-               Grid.Row="0"
-               Grid.Column="1" />
+               Grid.Column="3" />
+        <ComboBox  x:Name="Device" x:Uid="deviceSelector" Width = "190" HorizontalAlignment = "Left" SelectionChanged="Device_SelectionChanged" IsDropDownOpen="True" AllowDrop="True"  Header="Device" PlaceholderText="Pick a Device" Margin="0,1,0,1" Grid.Row="2" VerticalAlignment="Stretch" Grid.ColumnSpan="3">
+            <x:String>Intel CPU</x:String>
+            <x:String>Intel iGPU</x:String>
+            <x:String>Intel VPU</x:String>
```

Comment:
suggestion: create an enum in your .cs and set it as source to your Combobox the result of Enum.GetValues(typeof(<your enum>)) to avoid a switch on a redundant string value

Reply:
Fixed